lib/notification.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var service = require('./service').service;
+
+module.exports = {
+  create: function(movies, message) {
+    var twilioNumber = process.env.TWILIO_NUMBER;
```

Comment:
I don't think we should read environment variables after the application has finished initializing. I believe we should read them (and verify the variables are set) when the application starts. If they're not set, we should refuse to start with an informative error message.